diff --git a/dist/index.js b/dist/index.js
index e76d27a89367083f95854fc3b72f8eed88e5a5f6..47c3d5ce5605e02048102a123002d341161017db 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -57,7 +57,6 @@ function esbuildPluginPino({
       const customEntrypoints = {
         "thread-stream-worker": path.join(threadStream, "lib/worker.js"),
         "pino-worker": path.join(pino, "lib/worker.js"),
-        "pino-pipeline-worker": path.join(pino, "lib/worker-pipeline.js"),
         "pino-file": path.join(pino, "file.js")
       };
       const transportsEntrypoints = Object.fromEntries(
diff --git a/dist/index.mjs b/dist/index.mjs
index ffa9c33476053ae103e6c1ca4f42269d1c58f19f..5b9258a6e808c81cac5f374d45343f44a8cbfe2a 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -54,7 +54,6 @@ function esbuildPluginPino({
       const customEntrypoints = {
         "thread-stream-worker": path.join(threadStream, "lib/worker.js"),
         "pino-worker": path.join(pino, "lib/worker.js"),
-        "pino-pipeline-worker": path.join(pino, "lib/worker-pipeline.js"),
         "pino-file": path.join(pino, "file.js")
       };
       const transportsEntrypoints = Object.fromEntries(
